packages/story-editor/src/app/api/apiProvider.js

```diff
@@ -53,9 +53,11 @@ const STORY_FIELDS = [
   'permalink_template',
   'style_presets',
   'password',
+  'story-tags',
 ].join(',');
 
-const STORY_EMBED = 'wp:featuredmedia,wp:lockuser,author,wp:publisherlogo';
+const STORY_EMBED =
+  'wp:featuredmedia,wp:lockuser,author,wp:publisherlogo,wp:term';
```

Comment:
I think we also need to update that here:

https://github.com/google/web-stories-wp/blob/9976e33b321b60aea81b219599b97f53fc83854c/includes/templates/admin/edit-story.php#L45